public_html/assets/js/nf-core-schema-builder.js

```diff
@@ -1068,7 +1166,7 @@ function generate_param_row(id, param){
         <div class="col-auto align-self-center schema_row_grabber border-right">
             <i class="fas fa-grip-vertical"></i>
         </div>
-        <button class="col-auto align-self-center param_fa_icon ">`+fa_icon+`</button>
+        <button class="col-auto align-self-center param_fa_icon " title="Select icon" data-toggle="tooltip">`+fa_icon+`</button>
```

Comment:
We need to initialise these tooltips when new rows are added to the schema. I thought this was broken at first as I was starting with an empty schema and adding stuff.

Just needs a `row.find('[data-toggle="tooltip"]').tooltip()` somewhere.. (Let me know if you can't find it right away and I can take a look).

Reply:
Is it okay if I put that into the `init_group_sortable()` function? Would be called at the right time, but also a bit of a function overload...

Reply:
Hmm, I think that will not work for when new rows are added though (only groups). I would probably add it to the row / group generation code, ie. [here](https://github.com/nf-core/nf-co.re/blob/7c782c12fd6f8fa231970b30e5c5c3b9c2c84b48/public_html/assets/js/nf-core-schema-builder.js#L169) and [here](https://github.com/nf-core/nf-co.re/blob/7c782c12fd6f8fa231970b30e5c5c3b9c2c84b48/public_html/assets/js/nf-core-schema-builder.js#L193).

I guess it should just be something like this?

```javascript
param_row.prependTo('#schema-builder').find('.param_id').select();
param_row.tooltip();
```

Reply:
done 0ccb5b9